packages/x-charts/src/ChartsSurface.tsx

```diff
@@ -34,18 +38,50 @@ export const ChartsSurface = React.forwardRef<SVGSVGElement, ChartsSurfaceProps>
       viewBox,
       disableAxisListener = false,
       className,
+      onClick,
       ...other
     } = props;
     const svgView = { width, height, x: 0, y: 0, ...viewBox };
 
     useAxisEvents(disableAxisListener);
 
+    const { axis, item } = React.useContext(InteractionContext);
+    const series = React.useContext(SeriesContext);
+
+    const onLineClick = (event: React.MouseEvent<SVGSVGElement, MouseEvent>) => {
+      const itemData = onClick
```

Comment:
What would you think about moving this into the `useAxisEvent` with another `useEffect` such that when the charts does not require such event because `disableAxisListener=true` it just get skipped

```jsx
React.useEffect(() => {
    const element = svgRef.current;
    if (element === null || disableAxisListener) {
      return () => {};
    }
    
    const handleOnClick => { ... }
    
    element.addEventListener('click', handleOnClick)

    return () => {    element.removeEventListener('click', handleOnClick) }

    }, [axis, item, series, svgRef, disableAxisListener, onClick])
```

If you want something that should handle line, bar and scatter at the same time, you can have a look at the [axis tooltip](https://github.com/mui/mui-x/blob/master/packages/x-charts/src/ChartsTooltip/ChartsAxisTooltipContent.tsx/#L110-L127) that does a similar job of gathering information from different series